example/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  xbeach-vis:
+    image: deltares/docker-visualization-xbeach
+    depends_on:
+      - xbeach-run
+    ports:
+      - "8888:8888"
+  xbeach-run:
+    image: deltares/xbeach
```

Comment:
Kunnen we bij de xbeach-run niet ook al een port open zetten:
ports: 
  -53606:53606
Zodat het ook op windows 7 werkt? Samen met een aanpassing in het notebook?